src/savegame_json.cpp

```diff
@@ -1455,7 +1459,6 @@ void vehicle::deserialize(JsonIn &jsin)
     data.read("turn_dir", turn_dir);
     data.read("velocity", velocity);
     data.read("cruise_velocity", cruise_velocity);
-    data.read("music_id", music_id);
```

Comment:
The vehicle class still contains this member, but it's now unused, why not remove it?